guide/src/main/scala/io/udash/web/guide/views/ext/demo/BootstrapDemos.scala

```diff
@@ -444,7 +444,7 @@ object BootstrapDemos extends CrossLogging with CssView {
     }
     object UserModel extends HasModelPropertyCreator[UserModel]
 
-    val user = ModelProperty.empty[UserModel]
+    val user = ModelProperty(null: UserModel) // initialized below
```

Comment:
Since "Setting Property value to `null` is highly discouraged. Please, define implicit `Blank` for your type." you should not do that in guide. Also since Property.empty is removed now, a brief section (e.g. in `FrontendPropertiesView`) describing how to properly use `blank` could be useful to the users that got used to `empty`.